src/mpol/precomposed.py

```diff
@@ -50,6 +50,11 @@ def __init__(
         self.icube = images.ImageCube(
             coords=self.coords, nchan=self.nchan, passthrough=True
         )
+        
+        self.pbcube = images.PBCorrectedCube(
```

Comment:
Rather than modifying SimpleNet, I think it might be better to subclass it as a new SimplePBNet and then make your modifications to that. I think that for learning purposes new users will want to start with literally the simplest thing possible, and the PB correction is a step up from that